planner/core/plan_to_pb.go

```diff
@@ -195,6 +195,73 @@ func findColumnInfoByID(infos []*model.ColumnInfo, id int64) *model.ColumnInfo {
 	return nil
 }
 
+// ToPB generates the pb structure.
+func (e *PhysicalExchangeSender) ToPB(ctx sessionctx.Context, storeType kv.StoreType) (*tipb.Executor, error) {
+	child, err := e.Children()[0].ToPB(ctx, kv.TiFlash)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	encodedTask := make([][]byte, 0, len(e.Tasks))
+
+	for _, task := range e.Tasks {
+		encodedStr, err := task.ToPB().Marshal()
+		if err != nil {
+			return nil, errors.Trace(err)
+		}
+		encodedTask = append(encodedTask, encodedStr)
+	}
+
+	hashCols := make([]expression.Expression, 0, len(e.HashCols))
+	for _, col := range e.HashCols {
+		hashCols = append(hashCols, col)
+	}
+	hashColPb, err := expression.ExpressionsToPBList(ctx.GetSessionVars().StmtCtx, hashCols, ctx.GetClient())
```

Comment:
why here expressions, but Line:249 use FieldType?

Reply:
Here requires some columns, the line 249 only requires field type.